src/components/RsvpButton/index.js

```diff
@@ -59,15 +66,22 @@ class RsvpButton extends React.Component {
 
   handleClose() {
     this.setState({
-      visible: false,
+      modalVisible: false,
     });
   }
 
-  handleCloseOnSubmit() {
+  handleSuccess() {
+    this.setState({
+      formVisible: false,
+      successVisible: true
```

Comment:
aren't these always going to be opposites? I don't think you need both. ie if `success === true` show RsvpSuccess otherwise show form

Reply:
and you can just use `confirmed` which we already have in state